config/locales/devise.sv.yml

```diff
@@ -0,0 +1,63 @@
+# if me: DO NOT modify this file. if you need to add translations, add them to sv.yml
+# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
+
+sv:
+  devise:
+    confirmations:
+      confirmed: "Din e-postadress har bekräftats."
+      send_instructions: "Du kommer att få ett mail med instruktioner för hur du bekräftar din e-postadress om några minuter."
+      send_paranoid_instructions: "Om din e-postadress finns i vår databas får du ett mail med instruktioner för hur du bekräftar din e-postadress om några minuter."
+    failure:
+      already_authenticated: "Du är redan inloggad."
+      inactive: "Ditt konto är inte aktiverat än."
+      invalid: "Ogiltigt %{authentication_keys} eller lösenord."
+      locked: "Ditt konto är låst."
+      last_attempt: "Du har ytterligare ett försök innan ditt konto är låst."
+      not_found_in_database: "Ogiltigt %{authentication_keys} eller lösenord."
+      timeout: "Din session löpte ut. Vänligen logga in igen för att fortsätta."
+      unauthenticated: "Du måste logga in eller registrera dig innan du fortsätter."
+      unconfirmed: "Du måste bekräfta din e-postadress innan du fortsätter."
+    mailer:
+      confirmation_instructions:
+        subject: "Bekräftelsesinstruktioner"
+      reset_password_instructions:
+        subject: "Återställ lösenordsinstruktioner"
+      unlock_instructions:
+        subject: "Lås upp instruktionerna"
+      password_change:
+        subject: "Lösenordet ändrat"
+    omniauth_callbacks:
+      failure: "Det gick inte att autentisera dig från %{kind} eftersom \"%{reason}\"."
+      success: "Det gick att autentisera från %{kind} konto."
+    passwords:
+      no_token: "Du kan inte komma åt den här sidan utan att komma från ett lösenordsåterställnings-meddelande. Om du kommer från ett lösenordsåterställt e-postmeddelande, var vänlig och se till att du använde hela webbadressen."
+      send_instructions: "Du får ett mail med instruktioner om hur du återställer ditt lösenord om några minuter."
+      send_paranoid_instructions: "Om din e-postadress finns i vår databas, får du en länk för lösenordsåterställning på din e-postadress om några minuter."
+      updated: "Ditt lösenord har ändrats framgångsrikt. Du är nu inloggad."
+      updated_not_active: "Ditt lösenord har ändrats framgångsrikt."
+    registrations:
+      destroyed: "Adjö! Ditt konto har blivit avstängt. Vi hoppas att vi ses snart igen."
```

Comment:
"Adjö!" can be quite aggressive. Especially with a bang. "Hejdå!" is more neutral.